### ISSUE TITLE
Remove the deployed application from AWS

### ISSUE THREAD

**Issue by tanaydpatel** [serverless-nextjs/serverless-next.js#987]
I deployed the next application on AWS with `npx serverless`. 
But when I performed `serverless remove` it didn't removed the entry from cloud-front (disabled it) and I was also not able to remove the Lambda functions.

How to remove the deployed application ?

**Comment by davegariepy** [comment]
@tanaydpatel I came across this yesterday. I think it happens because the lambda@edge functions are associated with a cloudfront distribution and when you remove the stack the distribution takes a few minutes to delete.

I'm using CDK and ended up manually deleting the stack in the AWS Cloudformation console. I checked the lambdas after a few minutes in the console and then manually deleted those. Not ideal, but that's how I solved it.

Heres a link for your reference to the lambda@edge functions: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-edge-delete-replicas.html
